tardis/montecarlo/base.py

```diff
@@ -327,45 +327,6 @@ def output_nu(self):
     def output_energy(self):
         return u.Quantity(self._output_energy, u.erg)
 
-    @property
-    def virtual_packet_nu(self):
-        try:
-            return u.Quantity(self.virt_packet_nus, u.Hz)
-        except AttributeError:
-            warnings.warn(
-                "MontecarloRunner.virtual_packet_nu:"
-                "compile with --with-vpacket-logging"
-                "to access this property",
-                UserWarning,
-            )
-            return None
-
```

Comment:
These properties should not be removed. Instead, the warning should be rewritten to mention the required configuration setting instead of the compile flag. See the Important section here: https://tardis-sn.github.io/tardis/using/gui/sdec_plot.html?highlight=logging

Reply:
I had my university mid-semester exams this week so I apologize for the delay to make the respective changes. I have revised the warning of these properties.